frontend/src/views/mapper/concept/ConceptLink.js

```diff
@@ -3,8 +3,11 @@ import { withStyles } from '@material-ui/core/styles'
 
 const defaultAnchor = { x: 0.5, y: 0.5 }
 
+// TODO turn these into functional components
+
 export default class ConceptLink extends Component {
-  componentWillMount() {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillMount() {
```

Comment:
Should avoid using legacy code. Functions componentWillMount and componentWillReceiveProps could be done by a useEffect.